articles/model-anatomy.adoc

```diff
@@ -0,0 +1,678 @@
+include::../shared/issue-footnotes.adoc[]
+:serialization: ../serialization/serialization
+:m3: ../metametamodel/metametamodel
+:refarch: ../reference-architecture/reference-architecture
+:roadmap: ../roadmap/roadmap
+= Anatomy of a LionWeb Model
+:toc: preamble
+:toclevels: 3
+:sectnums:
+
+This article walks us through a LionWeb model:
+What it is, what's inside, how the inside looks, what the contents mean, and why they are that way.
+It is meant as an introduction to some of the details of LionWeb.
+We don't assume any prior knowledge of LionWeb; some experience with general modeling ideas is helpful.
+
+== Introduction
+The fundamental world view in LionWeb is "everything is a model".
+This means we express everything we talk about in this context in the form of models.
+
+But what is a model?
+STOP -- don't run away!
+This question is infamous in the modeling community -- but we have to answer it given our world view as stated above.
+We deliberately don't use that term in our specifications, as it's so overloaded and ambiguous.
+For our purposes, we can always replace "model" with "a bunch of nodes".
+Then the question transforms to: What is a node?
+
+[[notation]]
+== Notation used in this article and diagrams
```

Comment:
I would expect the "Nodes" paragraph at this point after the question What is nodes. Also I agree I am not sure what the "notations" paragraph gains at this point.